src/backend/base/langflow/components/jigsawstack/vocr.py

```
@@ -0,0 +1,107 @@
+                
+                params["page_range"] = [self.page_range_start, self.page_range_end]
+            
+            # Call web scraping
+            response = client.vision.vocr(params)
+            
+            if not response.get("success", False):
+                raise ValueError("JigsawStack API returned unsuccessful response")
+            
+            return Data(data=response)
+            
+        except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)